src/Microsoft.Azure.EventHubs/Primitives/EventHubsConnectionStringBuilder.cs

```diff
@@ -42,23 +43,26 @@ public class EventHubsConnectionStringBuilder
         const char KeyValuePairDelimiter = ';';
 
         /// <summary>
-        /// Build a connection string consumable by <see cref="EventHubClient.Create(string)"/>
+        /// Build a connection string consumable by <see cref="EventHubClient.CreateFromConnectionString(string)"/>
         /// </summary>
         /// <param name="namespaceName">Namespace name (the dns suffix, ex: .servicebus.windows.net, is not required)</param>
         /// <param name="entityPath">Entity path. For eventHubs case specify eventHub name.</param>
         /// <param name="sharedAccessKeyName">Shared Access Key name</param>
         /// <param name="sharedAccessKey">Shared Access Key</param>
         public EventHubsConnectionStringBuilder(string namespaceName, string entityPath, string sharedAccessKeyName, string sharedAccessKey)
-            : this(namespaceName, entityPath, sharedAccessKeyName, sharedAccessKey, DefaultOperationTimeout)
+            : this(namespaceName, entityPath, sharedAccessKeyName, sharedAccessKey, DefaultSbDomain)
```

Comment:
I know all of the namespaces are ".servicebus.windows.net", but this is subject to change in the future for Event Hubs. Can we change this to not include "Sb"?